### PR TITLE
Bump to ruby 2.3.6

### PR DESCRIPTION
**Pull request by jrafanie**
Bump to ruby 2.3.6

### CODE REVIEW

**Comment by juliancheal** [comment]
LGTM 👍

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: ruby
 rvm:
-- '2.3.3'
-- '2.4.1'
+- 2.3.6
+- 2.4.1
```
Comment (juliancheal) [inline .travis.yml]: Shouldn't this be `2.4.2`?

Comment (jrafanie) [reply .travis.yml]: Sorry, I'm not touching 2.4 in this PR beyond what YAML dump does where it strips the superfluous quotes here.  I'll come back to other versions and branches later.

**Comment by agrare** [comment]
@jrafanie can you rebase and repush?

**Comment by agrare** [comment]
@jrafanie fixed the merge conflict for you

**Pull request merged**